setup.rst

```diff
@@ -251,7 +251,7 @@ create new projects. If you use Composer, you need to tell the exact version:
 
 .. code-block:: terminal
 
-    $ composer create-project symfony/website-skeleton:^4.4 my_project_name
```

Comment:
There is indeed a confusion with the `require` argument syntax, could you please keep the minor version though?